### ISSUE TITLE
Version 4.7 shows this warning

### ISSUE THREAD

**Issue by wp4nuv** [cakephp/cakephp-codesniffer#387]
Related to #382 
Just created a project with CakePHP 4 and configured PHPStorm to use phpcs and this standard.  Issue 382 refers to a warning relating to slevomat/coding-standard.  Version 4.7 shows this error in PHPStorm:

`phpcs: ERROR: Referenced sniff "SlevomatCodingStandard.Arrays.TrailingArrayComma" does not exist`

I haven't begun coding on the project yet, so I'm unaware of any problems running phpcs in PHPStorm, but considering the implications in VSCode stated in 382, this may also impact that tool.

SlevomatCodingStandard installed is version 8.11.1

**Comment by wp4nuv** [comment]
Ok, just confirmed that phpcs fails to run, listing that same error. 
Interestingly, that sniff exists in the 8.11.1 version.  In PHPStorm, I have configured phpcs to use the CakePHP standard. 

```
devilbox@php-7.4.33 in /shared/httpd/api4.blnsoftware $ phpcbf --colors --standard=vendor/cakephp/cakephp-codesniffer/CakePHP -p src/
ERROR: Referenced sniff "SlevomatCodingStandard.Arrays.TrailingArrayComma" does not exist

Run "phpcbf --help" for usage information
```

**Comment by dereuromark** [comment]
This must be a misconfig of sorts, closing for now.
